.github/workflows/publish-technical-documentation-next.yml

```diff
@@ -0,0 +1,37 @@
+name: publish-technical-documentation-next
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "docs/sources/**"
+  workflow_dispatch:
+jobs:
+  sync:
+    if: github.repository == 'grafana/explore-logs'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Clone website-sync Action
+        # WEBSITE_SYNC_TOKEN is a fine-grained GitHub Personal Access Token that expires.
+        # It must be regenerated in the grafanabot GitHub account and requires a Grafana organization
+        # GitHub administrator to update the organization secret.
+        # The IT helpdesk can update the organization secret.
+        run: git clone --single-branch --no-tags --depth 1 -b master https://grafanabot:${{ secrets.WEBSITE_SYNC_TOKEN }}@github.com/grafana/website-sync ./.github/actions/website-sync
+
+      - name: Publish to website repository (next)
+        uses: ./.github/actions/website-sync
+        id: publish-next
+        with:
+          repository: grafana/website
```

Comment:
```suggestion
          repository: grafana/explore-logs
```

shouldn't it?

Reply:
Not sure. How does the action get's the content of `/docs/sources` repo?

Reply:
Ok, tried to understand the action a bit more. I think I got it now. Disregard my comments.

Reply:
Thanks for taking the time to understand the workflow :)